data/governments.txt

```diff
@@ -788,16 +927,46 @@ government "Kor Mereti"
 government "Kor Mereti (Hostile)"
 	"display name" "Kor Mereti"
 	swizzle 5
-	language "Korath"
+	language "Mereti"
 	"attitude toward"
 		"Wanderer" -.01
 		"Kor Mereti" -.01
 	"player reputation" -1000
 
+government "Kor Isaha"
+	"display name" "Korath"
+	swizzle 7
+	color "governments: Korath Exiles"
+	"crew attack" 1.4
+	"crew defense" 2.6
+	language "Korath"
+
+	"attitude toward"
+		"Kor Alaji" .01
+		"Kor Isaha" .01
```

Comment:
```suggestion
```
Unless there's a reason they have .01 attitude toward themselves.

Reply:
That should say "Korath" and I just fixed it in b516cf6. Thank you for finding this.